app/Listeners/SendFirstVoteMessage.php

```diff
@@ -30,37 +30,53 @@ public function handle(UserCastFirstVote $event)
         $payload = [
             'first_name' => $event->user->first_name,
             'birthdate_timestamp' => strtotime($event->user->birthdate), // Message Broker expects UNIX timestamp
-            'country_code' => $event->user->country_code,
+            'user_country' => $event->user->country_code,
             'candidate_id' => $event->candidate->id,
             'candidate_name' => $event->candidate->name,
+
+            'merge_vars' => [
+                'FNAME' => $event->user->first_name,
+                'CANDIDATE_NAME' => $event->candidate->name,
+                'CANDIDATE_LINK' => route('candidates.show', $event->candidate->slug),
+                'GENDER' => $event->candidate->gender,
+            ],
+
+            // This is always `en` because we don't translate Voting App & the
+            // experience should be consistent between app and messaging.
+            'user_language' => 'en',
         ];
 
-        // Send fields for domestic users
-        if ($event->user->country_code === 'US') {
+        // Send fields for SMS communication if provided.
+        if ($event->user->phone) {
             $payload['mobile'] = $event->user->phone;
             $payload['mobile_tags'] = [
                 env('APP_NAME_TAG', 'votingapp'),
                 $event->candidate->id,
                 'GENDER_'.$event->candidate->gender,
             ];
+
+            // Send Mobile Commons opt-in path for US users, and MGage ID for international users.
+            $payload['mobile_opt_in_path_id'] = $event->user->country_code === 'US' ? env('MC_OPT_IN_PATH') : env('MGAGE_ID');
```

Comment:
Okay, follow-up question – since we're sending `country_code` and the Message Broker is making the decision to use MobileCommons vs. MGage based on that, does it make sense for the application to just send all possible opt-in paths/list IDs identified by country code and let Message Broker decide between them?